tests/conftest.py

```diff
@@ -1,23 +0,0 @@
-# -*- coding: utf-8 -*-@pytest.fixture
-
-from __future__ import unicode_literals
-
-import functools
-
-import mock
-import pytest
-
-
-def autopatcher(request, target, **kwargs):
-    """Patch and cleanup automatically. Wraps :py:func:`mock.patch`."""
-    options = {'autospec': True}
-    options.update(kwargs)
-    patcher = mock.patch(target, **options)
-    obj = patcher.start()
-    request.addfinalizer(patcher.stop)
-    return obj
-
-
-@pytest.fixture
-def patch(request):
-    return functools.partial(autopatcher, request)
```

Comment:
I had to move the `tests/conftest.py` file to `tests/lti/conftest.py` to fix a Python import issue,